CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 1.9.3
+
+- Upgrade IotAppKit dependency from 9.2.0 to 9.6.0
```

Comment:
Instead of saying this, please include the exact changes included in this release as done for previous releases.

Reply:
done